nifi-docs/src/main/asciidoc/expression-language-guide.adoc

```diff
@@ -1644,9 +1644,9 @@ Divide. This is to preserve backwards compatibility and to not force rounding er
 
 *Description*: [.description]#Converts the Subject from a Base 10 number to a different Radix (or number base). An optional
 	second argument can be used to indicate the minimum number of characters to be used. If the converted value
-	has fewer than this number of characters, the number will be padded with leading zeroes.
+	has fewer than this number of characters, the number will be padded with leading zeroes.#
```

Comment:
I think that, instead of moving the # character, it'd be better to just remove the carriage return between this line and the next one, no? (same applies for fromRadix). Otherwise, the display is a bit weird in the doc:
https://github.com/Leah-Anderson/nifi/blob/a445ee4d141ea1218d3cf5c47abb5171c5890f43/nifi-docs/src/main/asciidoc/expression-language-guide.adoc#fromradix

Reply:
I was thinking the way it was made it seem like a call out to "hey this is important to note" but I can see it either way. I've pushed an update to remove the carriage return :)